app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -13,9 +13,11 @@ def index
   def create
     @planning_application = PlanningApplication.create(full_planning_application_params)
     if @planning_application.save!
-      render json: {"message": "Application created: id was #{@planning_application.reference}"}, status: 200
+      @planning_application.create_policy_evaluation
```

Comment:
This is a complete aside, but why do we have policy_evaluation as a `has_one`?

Reply:
From memory, I think it was the necessity to model the real-world application - ie, the requirement that a planning application is assessed against only one set of policy considerations.